internal/services/media/registration.go

```diff
@@ -67,7 +67,5 @@ func (r Registration) DataSources() []sdk.DataSource {
 }
 
 func (r Registration) Resources() []sdk.Resource {
-	return []sdk.Resource{
-		AccountFilterResource{},
-	}
+	return []sdk.Resource{}
```

Comment:
sorry for the confusion @magodo , I think we need to leave this as it is for now and not remove it,  since it wasn't feature flagged before 4.0 was released

Reply:
@catriona-m Oh.. I see. That makes sense now. I've reverted it back.